themes/powerline/powerline.base.sh

```diff
@@ -128,11 +168,16 @@ function __powerline_left_segment {
   IFS="${OLD_IFS}"
   local separator_char="${POWERLINE_LEFT_SEPARATOR}"
   local separator=""
+  local text_color="-"
+  if [[ -n "${params[2]}" ]]; then
+    text_color="${params[2]}"
+  fi
```

Comment:
Is it different from the following?

```suggestion
  local text_color=${params[2]:-"-"}
```